unstructured/partition/pdf.py

```diff
@@ -326,12 +326,16 @@ def _process_pdfminer_pages(
                 _text = clean_extra_whitespace(_text)
                 if _text.strip():
                     text_segments.append(_text)
-                    element = element_from_text(_text)
                     coordinate_system = PixelSpace(
                         width=width,
                         height=height,
                     )
                     points = ((x1, y1), (x1, y2), (x2, y2), (x2, y1))
+                    element = element_from_text(
+                        _text,
+                        coordinates=points,
+                        coordinate_system=coordinate_system,
+                    )
```

Comment:
@qued IIUC this change doesn't make a difference when partitioning with Chipper but I don't see the harm of making it. Let me know if you think differently.

Reply:
If we construct the element this way, should we get rid of the lines that follow that create coordinate metadata and store it in the metadata, since presumably it should already be there?

Reply:
Actually, I'm fine if we just merge this. I think there's some inconsistency around how we're handling coordinates vs other metadata, but I'll create an issue to track that.

Reply:
I made these changes locally and unit tests passed:
<img width="646" alt="change_havent_committed" src="https://github.com/Unstructured-IO/unstructured/assets/1435725/df360f9b-bcd2-4c5d-88e7-a7c1759e817e">

I didn't push them because I saw [your comment about creating a new issue](https://github.com/Unstructured-IO/unstructured/pull/913#discussion_r1259167066).